### ISSUE TITLE
Ignore historical builds

### ISSUE THREAD

**Issue by dennisdoomen** [dennisdoomen/Beacon#8]
If a historical build succeeded, but the last normal build failed, Beacon will report a success for that build configuration.

**Comment by gormac** [comment]
Is this about the TeamCity History Builds concept? See [History Builds](https://confluence.jetbrains.com/display/TCD10/History+Build)

**Comment by dennisdoomen** [comment]
No, it's about aligning the branch that Beacon should look at with what you see in TC. If a branch which hasn't had any changes for a while fails, TC doesn't count that as a real failure.
